policies/queries/elb/listeners_unused.sql

```diff
@@ -0,0 +1,12 @@
+insert into aws_policy_results
+select :'execution_time'                        as execution_time,
+       :'framework'                             as framework,
+       :'check_id'                              as check_id,
+       'Unused ELB listeners should be removed' as title,
+       account_id,
+       arn                                      as resource_id,
+       case
+           when length(load_balancer_arn) > 0 then 'pass'
+           else 'fail'
+           end                                  as status
+from aws_elbv2_target_groups
```

Comment:
What about the internal/hardcoded responses (for example redirects)...

Reply:
adding the check for aws_elbv2_listener_default_actions as well

Reply:
decided to remove